api/src/main/java/com/velocitypowered/api/newcommand/CommandManager.java

```diff
@@ -0,0 +1,55 @@
+package com.velocitypowered.api.newcommand;
+
+import com.velocitypowered.api.command.CommandSource;
+import com.velocitypowered.api.event.command.CommandExecuteEvent;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Registers and executes commands.
+ */
+public interface CommandManager {
+
+    /**
+     * Registers the specified command.
+     *
+     * @param command the command to register
+     */
+    void register(Command<?> command);
+
+    /**
+     * Unregisters the specified command.
+     *
+     * @param command the command to unregister
+     */
+    void unregister(Command<?> command);
+
+    /**
+     * Unregisters the command with the specified alias.
+     *
+     * @param alias the alias of the command
+     */
+    // TODO Previous javadoc is ambiguous, does this unregister all the aliases of the command,
+    // or just the given alias?
```

Comment:
The previous Javadoc description should have read `Unregisters the specified alias from the command manager`.